doc/source/release/1.17.0-notes.rst

```diff
@@ -170,22 +170,18 @@ The functions `load`, and ``lib.format.read_array`` take an
 ``allow_pickle`` keyword which now defaults to ``False`` in response to
 `CVE-2019-6446 <https://nvd.nist.gov/vuln/detail/CVE-2019-6446>`_.
 
-
-.. currentmodule:: numpy.random.mtrand
```

Comment:
Would changing this to `.. currentmodule:: numpy.random` have been enough to fix the links?

Reply:
That seems to be enough.